services/venice-server/src/main/java/com/linkedin/venice/server/VeniceServer.java

```diff
@@ -245,6 +234,102 @@ private List<AbstractVeniceService> createServices() {
     liveClusterConfigRepo = veniceMetadataRepositoryBuilder.getLiveClusterConfigRepo();
     readOnlyZKSharedSchemaRepository = veniceMetadataRepositoryBuilder.getReadOnlyZKSharedSchemaRepository();
 
+    List<VeniceSystemStoreType> schemaStoresForWritableComponents = Arrays.asList(
+        // VeniceSystemStoreType.TEST_DUMMY,
+        VeniceSystemStoreType.PARTITION_STATE,
+        VeniceSystemStoreType.STORE_VERSION_STATE,
+        VeniceSystemStoreType.KAFKA_MESSAGE_ENVELOPE,
+        VeniceSystemStoreType.META_STORE,
```

Comment:
There are duplicate KME and meta system store schema check at line395, which could be removed?
```
if (serverConfig.isSchemaPresenceCheckEnabled()) {
      kafkaMessageEnvelopeSchemaReader.ifPresent(
          schemaReader -> new SchemaPresenceChecker(schemaReader, AvroProtocolDefinition.KAFKA_MESSAGE_ENVELOPE)
              .verifySchemaVersionPresentOrExit());
      metaSystemStoreSchemaReader.ifPresent(
          schemaReader -> new SchemaPresenceChecker(schemaReader, AvroProtocolDefinition.METADATA_SYSTEM_SCHEMA_STORE)
              .verifySchemaVersionPresentOrExit());
    }
```

Reply:
Oh, I might have missed this. Will take a look